pandas/core/window.py

```diff
@@ -899,7 +899,53 @@ def skew(self, **kwargs):
         return self._apply('roll_skew', 'skew',
                            check_minp=_require_min_periods(3), **kwargs)
 
-    _shared_docs['kurt'] = """Unbiased %(name)s kurtosis"""
+    _shared_docs['kurt'] = dedent("""
+    Calculate unbiased %(name)s kurtosis.
+
+    This function uses Fisher's definition of kurtosis without bias.
+
+    Parameters
+    ----------
+    kwargs : Under Review
+
+    Returns
+    -------
+    Series or DataFrame (matches input)
+        Like-indexed object containing the result of function application
+
+    See Also
+    --------
+    pandas.Series.%(name)s
+    pandas.DataFrame.%(name)s
+    pandas.Series.kurtosis
+    pandas.DataFrame.kurtosis
+    scipy.stats.skew
+    scipy.stats.kurtosis
+
+    Notes
+    -----
+    A minimum of 4 periods is required for the rolling calculation.
+
+    Examples
+    --------
+    The below example will show a rolling calculation with a window size of
+    four matching the equivalent function call using `scipy.stats`.
+
+    >>> arr = [1, 2, 3, 4, 999]
+    >>> import scipy.stats
+    >>> print("{0:.6f}".format(scipy.stats.kurtosis(arr[:-1], bias=False)))
+    -1.200000
```

Comment:
Fair enough. Let me mess around with iPython - I think we can use the %precision directive here behind the scenes

Reply:
Hmm, the annoying thing here is that scipy prints 1.99999999999, as otherwise `1.200...` would have worked (using an ellipsis to match anything on that line)

Yeah, not sure there is a better solution in  this case

Reply:
Yea I was hoping that setting `np.print_options` would help for SciPy but no luck. I kind of wonder if globally it wouldn't be preferable to just set this to 6 across all docstrings for consistency

Reply:
+1 on setting it globally

Reply:
Looked into it and I think that would unfortunately be pretty difficult for scalar values. `np.set_printoptions` may be worthwhile for arrays, but as far as this specific example is concerned the formatting of scalar values from what I can tell is left to interpreters like iPython. I looked at DocTests' documentation but didn't see anything there that would easily allow us to enforce that option during its testing